src/total-value-and-liquidity/total-value-and-liquidity.js

```diff
@@ -0,0 +1,130 @@
+const ethers = require('ethers');
+const BigNumber = require('bignumber.js');
+const { getJsonRpcUrl } = require('forta-agent');
+const RollingMathLib = require('rolling-math');
+
+// get createAlert and dataFields
+const { createAlert, dataFields } = require('./common');
+
+// load required shared types
+const contractAddresses = require('../../contract-addresses.json');
+
+const { LendingPool: LendingPoolAddr, ProtocolDataProvider: DataProviderAddr } = contractAddresses;
+const { abi: DataAbi } = require('../../interfaces/AaveProtocolDataProvider.json');
+const { abi: LendingPoolAbi } = require('../../interfaces/ILendingPool.json');
+
+// get config settings
+const {
+  totalValueAndLiquidity: Config,
+} = require('../../agent-config.json');
```

Comment:
how did the linter break this out onto 3 seperate lines when line 14 is longer?

Reply:
dunno XD /shrug